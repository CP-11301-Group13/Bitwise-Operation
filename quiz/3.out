48982 38048 44582 25713 27475 8679 56042 64494 58182 16491 41094 30759 45962 43590 57857 55587 31901 61079 1941 52423 47995 19782 26151 52538 25887 13540 59585 54417 21256 25107