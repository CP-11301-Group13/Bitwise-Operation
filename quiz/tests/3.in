30 2
61273 24992 43913 37332 40540 33981 31418 65211 48153 4254 41001 53901 60458 43545 47108 30348 54119 47981 3429 13117 61150 5913 39309 14282 38223 49585 45620 29028 23554 38988