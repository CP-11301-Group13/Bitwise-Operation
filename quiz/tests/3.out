41764 41764